lib/jsdom/living/attributes.js

```diff
@@ -375,7 +374,9 @@ exports.setAttributeValue = function (element, localName, value, prefix, namespa
 
   const attribute = exports.getAttributeByNameNS(element, namespace, localName);
   if (attribute === null) {
-    const newAttribute = attrGenerated.createImpl([], { namespace, namespacePrefix: prefix, localName, value });
+    const newAttribute = element._ownerDocument._createNode(NODE_TYPE.ATTRIBUTE_NODE, [], {
```

Comment:
Again, the old code should still work here. There's no advantage to using _createNode (here and below).